docs/src/content/docs/user-roles/dac-authorization.mdx

```diff
@@ -0,0 +1,110 @@
+---
+title: DAC authorization
+description: How to give a user access based on a Data Access Request approval
+tableOfContents: true
+next: false
+banner:
+  content: |
+    These changes are in develop and will be part of an upcoming stable release.
+---
+
+import { Badge } from '@astrojs/starlight/components';
+
+## Data Access Committee process
+
+It is assumed that before a site admin can grant access to an external user that a Data Access Committee (DAC) process has been undertaken outside of CanDIG. The result of this process would then be communicated to the site admin so that they can use these details to grant the user access to the appropriate datasets for the specified amount of time. Once granted access, a user will have full access to all donor level data, including both clinical and genomic data. 
```

Comment:
let's not say "external user" because that is not well-defined at the moment (external to the local CanDIG instuttion? to the MOHCCN network? ). Proposal for this paragraph:

A Data Access Committee can approve access for a given user to specific data sets. Once a DAC decision is communicated to the site admin, this documentation describes how to implement DAC Authorizations. nce granted access, a user will have full access to all donor level data, including both clinical and genomic data.